src/components/featured-item/featured-item.html.twig

```diff
@@ -8,9 +8,6 @@
   - "link" (object) (default: {}): object of type Link
   - "media_container" (object) (default: {}): object of type Media container
   - "position" (object) (default: 'left'): Media position
-  - "footer_description" (string) (default: ''): short description in the footer
```

Comment:
but those are still there, you only removed this docs from the template but the parameters are still in place and used, then i guess that in another pr all of this will be cleaned up, this is a bit inconsistent here, here, imagine if we release this and not the pull request where the parameters will be effectively removed.

Reply:
those are already in deprecated part below, that's why i though they are reduntant